ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -265,31 +266,49 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                         <div className='columns small-1' />
                         <div className='columns small-3'>NAME</div>
                         <div className='columns small-2'>NAMESPACE</div>
-                        <div className='columns small-2'>STARTED</div>
-                        <div className='columns small-2'>FINISHED</div>
-                        <div className='columns small-2'>DURATION</div>
+                        <div className='columns small-1'>STARTED</div>
+                        <div className='columns small-1'>FINISHED</div>
```

Comment:
Not a fan of these (`Started` and `Finished`) columns going from 2 to 1. Even on my large laptop screen there's not enough space to display the full string. I think `Duration` is fine to go from 2 to 1 since it's expected to be a smaller string. We could probably get away with taking another one from `Name` (3 to 2)? Can you play around and find something that doesn't hide the strings as much as possible?

Reply:
Will do!

Reply:
On all the example workflows I've used / seen so far, the namespace is `argo`. Is it pretty typical for the namespace to be short like this, or do a lot of users have typically long namespaces?

Reply:
> Is it pretty typical for the namespace to be short like this, or do a lot of users have typically long namespaces?

I would say it's more typical to have long namespace names. I don't think `columns 1` would cut it

Reply:
Gotcha. There is a lot of wasted space in two places: the first unnamed column with the status icon, and the labels column when labels are hidden. What do you think of having a popup or tooltip upon clicking "SHOW" instead of displaying them in the column itself?

Reply:
Another option I just thought of is displaying all the labels as a new row underneath the desired row upon clicking SHOW. I'll create it real quick and include a screenshot.

Reply:
<img width="1172" alt="Screen Shot 2020-06-02 at 1 10 26 PM" src="https://user-images.githubusercontent.com/6250584/83564969-6e5e9080-a4d2-11ea-8ded-ca31b5af5f2d.png">

Reply:
Looks good, but can we add a way to emphasize that it's still part of the row above? Perhaps some sort of "drawer" where the background of the labels is a dimmer shade of white. Do you know what I mean?

Reply:
Absolutely! Good idea

Reply:
Something like this? 

<img width="1187" alt="Screen Shot 2020-06-02 at 2 25 35 PM" src="https://user-images.githubusercontent.com/6250584/83571594-ee89f380-a4dc-11ea-9e6f-cc137db4aebc.png">

Reply:
This looks amazing! In my opinion, I'd prefer a lighter background as the contrast with the blue of the labels is not too great. What do you think?

Now I'm thinking why stop with labels? This is a great UX improvement and I'm thinking we can fill this drawer with all sorts of info about the Workflow. Take a look at this tab in "Workflow Details". I'm thinking we can add "message", "resources duration", and "conditions" here, among others. Then instead of the column being called "labels" we can just have it be "More details".

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/614838/83572857-2abe5380-a4df-11ea-96f5-052d94746d6c.png">

What do you think?

Reply:
Thanks! I agree with the poor contrast of the teal and dark gray, I'll play around with the colors and see what works better. 

That's a great idea! I'll start working on it now; I'm thinking that probably falls out of the scope of this PR though so should I open a new one?

Reply:
> I'm thinking that probably falls out of the scope of this PR though so should I open a new one?

Sure thing a new one is totally fine. Just make sure it's the next one you work on since it's going to overwrrite some of the work you're doing here. Let me know when you've found some colors you find and I'll give this another review.

Reply:
Sounds good. Here are the improved colors: 

<img width="1172" alt="Screen Shot 2020-06-02 at 2 57 16 PM" src="https://user-images.githubusercontent.com/6250584/83573996-5b06f180-a4e1-11ea-984a-48a5a6169448.png">